### ISSUE TITLE
Getting an "Missing end time." error

### ISSUE THREAD

**Issue by alexmlong** [jrmdayn/googleapis-batcher#4]
Hey, thanks for making this useful package. I just wanted to let you know that I'm trying to use it and hitting this error:

```
  errors: [
    {
      domain: 'global',
      reason: 'required',
      message: 'Missing end time.'
    }
  ]
```

I tried erasing my node modules and package-lock, then reinstalling everything, and I still got this error. I only get it when trying to insert multiple calendar events in a batch. When I try to insert a single event it works fine.

If you have any ideas about what's going on here that'd be great. I've checked the data format and despite the error, the "end time" is provided and is in the valid format.

**Comment by jrmdayn** [comment]
Hey,

I published v0.4.0 with `debug` logs. Could you run your code with `DEBUG=googleapis-batcher` and share the logs with me (you should erase sensitive info like tokens, content etc..)?

Thanks

**Comment by juanmiguelbesada** [comment]
Hey guys!

I was facing this issue as well, and was able to spot the bug. I created a PR to fix it.

@jrmdayn If you can take a look when you can 🙏

**Comment by jrmdayn** [comment]
Thanks @juanmiguelbesada for looking into this. Indeed there were some issues with the body - thanks for fixing them.

I have released v0.5.0 let me know if it works for you guys on your end.

🙏🙏🙏
